packages/web-ui/src/lab/TextLink/TextLink.docs.mdx

```diff
@@ -0,0 +1,109 @@
+import { Meta, Canvas, ArgTypes, Description } from '@storybook/blocks';
+import * as Stories from './TextLink.stories';
+
+<Meta of={Stories} />
+
+# Link
+
+<Description />
+
+<Canvas of={Stories.Workshop} />
+
+## Usage
+
+This `TextLink` component is part of the lab submodule, a way for us to make new
+components available prior to an official major release. This helps us bed in
+new components before introducing breaking changes, and to avoid blocking the
+availability of these components for designers.
+
+There are 2 options for importing this component.
+
+1. Import directly from the `lab` directory.
+
+```tsx
+import { Text } from '@utilitywarehouse/web-ui';
+import { TextLink } from '@utilitywarehouse/web-ui/dist/lab';
+
+[...]
+
+<Text>
+  <TextLink>TextLink</TextLink>
+</Text>
+```
+
+2. Import `Lab` directly from Web UI.
+
+```tsx
+import { Text, Lab } from '@utilitywarehouse/web-ui';
+
+[...]
+
+<Text>
+  <Lab.TextLink>TextLink</Lab.TextLink>
+</Text>
+// or
+const { TextLink } = Lab
+<Text>
+  <TextLink>TextLink</TextLink>
+</Text>
+```
+
+## Semantic HTML
+
+A semantic HTML `a` is rendered by default, this cannot be changed due to the
+expectations on links within blocks of text.
+
+## Size
+
+The `TextLink` component must be wrapped in a `Text` component, and will
+inherit it's styles, so it's size will can be controlled by the contianing
```

Comment:
Typo at the end of this line: "contianing".

Reply:
good catch, thanks James 🙇🏻